### ISSUE TITLE
jsx-indent false positive 

### ISSUE THREAD

**Issue by pfarnach** [yannickcr/eslint-plugin-react#1085]
Using `v6.10.0`, I'm getting the following errors from this code:

```
import React from 'react';

const groups = [
  [
    <span />,
    <span />,
    <span />
  ],
  [
    <span />
  ]
];
```

```
6:5  error  Expected indentation of 2 space characters but found 4  react/jsx-indent
7:5  error  Expected indentation of 2 space characters but found 4  react/jsx-indent
```

If I "fix" these errors by un-indenting lines 6 and 7 (but not 5), I get generic indent errors:

```
6:3  error  Expected indentation of 4 spaces but found 2  indent
7:3  error  Expected indentation of 4 spaces but found 2  indent
```

**Comment by jseminck** [comment]
Could not reproduce this on master. Tried with both 2 and 4 spaces as indent. @pfarnach can you check if this is still an issue with latest ?

**Comment by pfarnach** [comment]
I don't have access to that repo anymore and this may have been fixed by now so we can close.
